src/main/java/com/cen4010/bookstore/book/Book.java

```diff
@@ -69,49 +73,87 @@ public Book(UUID id, String ISBN, String name, String description, double price,
     this.copiesSold = copiesSold;
   }
 
-  public Book() {
```

Comment:
Why remove default constructor? IIRC it may be necessary for some functionalities of Spring JPA

Reply:
So I actually moved the public Book() constructor to the top before the other two constructors.  I guess my OCD kicked in just thought it was better on top.

Reply:
My bad, just misread the code. No problem here.